main.py

```
@@ -6,37 +6,26 @@
-        context (google.cloud.functions.Context): Metadata of triggering event.
-    Returns:
-        None; the output is written to Stackdriver Logging
-    """
-
-    # print('Event type: {}'.format(context.event_type))
-    # print('Bucket: {}'.format(event['bucket']))
-    # print('File: {}'.format(event['name']))
-    # print('Metageneration: {}'.format(event['metageneration']))
-    # print('Created: {}'.format(event['timeCreated']))
-    # print('Updated: {}'.format(event['updated']))
+def process_box_score(event: CloudEvent):
```

Ruff (0.11.9)

Missing return type annotation for public function `process_box_score`

Add return type annotation: `None`

(ANN201)